agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -168,7 +168,7 @@ class TemplateDB(Model):
     type: Optional[str] = Field(default="image")
     template_uri: Optional[str]
     tag_id: Optional[int]
-    name: str  # tag name of image
+    name: str = Field(unique=True)  # tag name of image
```

Comment:
why the change

Reply:
It's meant for the cloud version, since we pull from s3 which has no unique identifier except the name of the template.